docs/papers/squad-paper.md

```diff
@@ -1,56 +1,104 @@
+1. describe a general architecture for such a system, and 
+1. consider broader applications of the system.
 
 ## The Opportunity
 
-* Games are hard to design and develop. They require lots of
-  playtesting, the more people the better
-* The most popular games at the time of this writing are based on community
-  designed and developed mods
-  * MOBAs
-  * battle royales
-  * Hero shooters
-
-[Balancing games is super hard](https://www.gamasutra.com/blogs/DanFelder/20151012/251443/Design_101_Balancing_Games.php)
-"No matter how good you are at balancing, the collective intelligence of your
+### Decentralized game design has the potential to 10x outperform centralized game design.
+* Games are hard to design. 
+  * In theory:
+    * The design space is too large to be well-explored in a centralized way
+  * And in practice:
+    * The most popular games at the time of this writing are based on community
+  designed and developed mods:
+      * MOBAs
+      * battle royales
+      * Hero shooters
+
+* [Balancing games is super hard.](https://www.gamasutra.com/blogs/DanFelder/20151012/251443/Design_101_Balancing_Games.php)
+  * In theory:
+    * "No matter how good you are at balancing, the collective intelligence of your
 players is going to outmatch you" - Dan Felder
+  * And in practice:
+    * Large game companies employ entire teams to balance their games and still often receive harsh criticism on balance
+
+### Decentralized game design is viable in the near future
+
+Even though today's decentralized systems are limited, 
+we believe a successful system for decentralized game design is possible in the near future,
+because it depends on simple principles: 
+1. Game design is relatively simple and harmless.
+1. Contribution and curation are sufficient for progress.
+
+#### Game design is relatively simple and harmless
+This is *not* intended to belittle the difficulty of good game design (see above). 
+We mean that relative to, for example, designing and building a smart energy grid, 
+designing a game is simpler, and that the consequences of the energy grid failing
+are much larger than a game design failing.
+
+#### Contribution and curation are sufficient for progress
+This principle says that a system that supports these two, relatively simple features has no theoretical achievement limit.
+
+*Contribution* means things can be added to the system.
+*Curation* means contributions can be grouped arbitrarily.
+
+Programs can be thought of as systems that use only these two properties, and so can games.
+
+We believe Squad only needs to do these two things well to succeed at decentralized game design and balance. 
 
 ## Cryptoeconomic Mechanism
+A decentralized system for contribution and curation must incentivize those two things.
 
-Design an efficient mechanism that rewards the collective intelligence
-fairly for their investment in and contributions to the design and
-development of video games.
```

Comment:
I think I still like this basic sentence, maybe could use some editing, but I don't want to lose this premise/goal of the mechanism

Reply:
Agreed, trying another version.